### ISSUE TITLE
Title element should be more descriptive to avoid confusion

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#70]
URL: https://hcdev.openplus.ca/fr/documentation/onglet-rechercher
User Agent: Chrome
Language: both

Issue Description: <title> element for all pages should be similar to the h1 of the page. Screen reader users are read the page title to understand where they are and provide context. 

Currently you have a Dashboard set to "username | RSAMS" which isn't very clear, Modify your profile is the same.
When you are within the Help page, when you click on a section "Dashboard" the page title is reflective of this, however, it should include something to indicate that you are within the Help section. This could lead to confusion as user may think they are on the actual "Dashboard" page. 

Steps to reproduce:
navigate to pages and look at the tab title or view the source code and search for the <title> element 

Expected results:
<title> elements should be similar to h1 and provide context

**Comment by ndolter** [comment]
@echouin Since I don't have access to the Help page in DEV, could I please request that you send me the page title element in both French and English from Prod?  This can be found in the html code by searching on <title>.
For example:
![image](https://user-images.githubusercontent.com/68117281/161132481-7db7a223-2762-4dab-a8a9-730875eb90b6.png)

**Comment by echouin** [comment]
@ndolter could we do an MS Teams call so that you can show me what you need?
OR I can add you to Prod?

**Comment by ndolter** [comment]
@echouin Sure, if you're available to call me now in Teams that would work.  I'll be signing out for the day at 4:00pm. In case we don't get to chat today, please feel free to reach out tomorrow morning.

**Comment by echouin** [comment]
<title>Recalls and Safety Alerts Help Guide - Canada.ca</title>

FR: <title>Page non trouvée – Rappels, avis et avis de sécurité – Canada.ca</title>

**Comment by ndolter** [comment]
Thanks @echouin. 

For the French Help guide title element it should be translated to match the English page 'Recalls and Safety Alerts Help Guide - Canada.ca. 

The following are the observations for the Dashboard page and the tabs within them.
![image](https://user-images.githubusercontent.com/68117281/161139539-5123edaa-7d3a-4d37-a50b-236e4644a287.png)

**Comment by echouin** [comment]
@ndolter I'm not quite sure I understand what is needed? Can you please let me know how I would correct this?

**Comment by ndolter** [comment]
@echouin Similar to the exercise we did yesterday when you shared your screen with me, you would have to do the following:
1. Access the html code (right click on the page and select Inspect)
2. Search on <title> via Ctl-F and make the updates
This will control what gets displayed and announced by the screen reader in the upper page tab
![image](https://user-images.githubusercontent.com/68117281/161294687-479bb1e8-ed16-4a17-9882-ef39be4dcdca.png)

**Comment by echouin** [comment]
This issue is not happening in prod:
![image](https://github.com/user-attachments/assets/8cbe268c-61a4-42b9-b945-050a051f1174)
